noSSO parameter on iOS and refactoring iOS project (#117)
okta/okta-react-native: README.md

signInWithBrowser({ idp: 'your_idp_here' });
```

> **Note**: If you want to get rid of sign in and sign out alert on iOS, then pass `noSSO` parameter. The cookies are not retained by browser, so after logout a user will be prompted to re-authenticate.

Reviewer: ```suggestion
**Note**: If you want to get rid of the system sign in and sign out alert on iOS, then pass the `noSSO` parameter when calling `signInWithBrowser`. The cookies will not be retained by the browser, so after logging out the user will be prompted to re-authenticate.
```

```javascript

> signInWithBrowser({ noSSO: 'true' });

Reviewer: Does `true` need to be a string, or can a boolean be supplied?

Author: Swift bridge supplies only string, so I am not sure whether JS converts boolean to string while invoking the methods. Need to double check.

Reviewer: In the past, I've implemented convenience wrappers in JS that captures the boolean "true" and passes along the string equivalent.

Having to specify the exact string value "true" (e.g. case sensitive, etc) feels a bit weird for something that looks like a boolean.

Author: This won't work in our case. Because bridge method supplies [[String: String]](https://github.com/okta/okta-react-native/blob/deabc77162dd07217848189564c6c6d0df426f57/ios/OktaSdkBridge.swift#L52) dictionary.

If we pass boolean the exception is thrown `Could not cast value of type '__NSCFBoolean' (0x7fff86d714b8) to 'NSString'`. 

We may try to refactor to `[String: Any]` but we'll have to convert to string anyway, because the method in OIDC [supplies Strings](https://github.com/okta/okta-oidc-ios/blob/3772438422cfc61b8e15a5bcf10ac6d4f53843aa/Sources/OktaOidc/iOS/OktaOidc%2BBrowserIOS.swift#L25). 

Note: The developer can convert boolean to string via the method `toString()` - `enableSSO.toString()`.

Reviewer: I more meant, at the JS level we can accept booleans, and then convert it to a string to pass to the Swift API.  In this way, the developer can use more sensible values, and is insulated from the specifics of the React bridge API.

Author: Done.
